server/db/migrations/20_create_grid_service_instance.rb

```diff
@@ -1,7 +1,12 @@
 class CreateGridServiceInstance < Mongodb::Migration
   def self.up
     GridServiceInstance.create_indexes
-    Container.unscoped.where(container_type: 'container').includes(:grid_service).each do |c|
+    migrate_containers('volume')
+    migrate_containers('container')
```

Comment:
Only stateful services have volume containers. And stateful service instances have always been scheduled to a node where volume container is living. So this should be safe.